Packages/Tracking/OpenXR/Runtime/Ultraleap.Tracking.OpenXR.asmdef

```diff
@@ -6,7 +6,8 @@
         "GUID:96aa6ba065960476598f8f643e7252b6",
         "GUID:cec9848704c49724192ff47f9b3e9156",
         "GUID:2ee0542102ac1084e8d9a910885c1e81",
-        "GUID:ba171b3dd2a51234ab864770f99741a5"
+        "GUID:ba171b3dd2a51234ab864770f99741a5",
+        "GUID:e40ba710768534012815d3193fa296cb"
```

Comment:
What's this new reference?

Reply:
This is Unity's XR Management package (should really be necessary for XR work anyway)